components/ResultsPageComponents/CommuteDistanceDistribution/CommuteDistanceDistributionChart.jsx

```diff
@@ -17,27 +19,39 @@ export default function CommuteDistanceDistributionChart({ title, data }) {
   const highChartsOptions = {
     title: {
       text: title,
+      style: { color: "#022640" },
     },
     chart: {
       type: "column",
+      height: 400,
     },
     colors: ["#044B7F"],
     tooltip: {
-      pointFormat: "{point.y}%",
+      pointFormat: "{point.y}",
     },
     xAxis: {
       categories: Object.keys(data),
+      title: {
+        text: "Distance",
+      },
+      labels: {
+        formatter: function () {
+          return this.value + " km";
+        },
+      },
     },
     yAxis: {
       min: 0,
       title: {
-        text: "distance",
+        text: "No. of people",
       },
     },
-    series: [{ name: "Distance", data: chartData }],
+    series: [{ showInLegend: false, name: "Distance", data: chartData }],
   };
 
   return (
-    <HighchartsReact highcharts={Highcharts} options={highChartsOptions} />
+    <Box maxWidth="634px" width="100%">
```

Comment:
Hi @RCode-Blue , just a question - what is the reason you got back to Box rather than Flex? It gets cropped for me as well (same as what @hqtan got).

Reply:
Hey @OlaShabalina !

Yea, I noticed the cropping too when I was viewing preview deployment in responsive settings on my firefox browser

![image](https://user-images.githubusercontent.com/4408259/184560040-e7594bb0-6274-4fa7-a8d9-6fbb9d52f976.png)

But, it's interesting that when I view the preview deployment on my mobile phone, the cropping doesn't occur! (see my screenshot in [this comment](https://github.com/CodeforAustralia/council-emissions-calculator/pull/177#issuecomment-1214479884))

Reply:
It didn't seem to make a difference in my tests, and given that <Flex> is basically <Box> with a flex view, and it didn't seem like we needed it, It seemed unnecessary. I'm totally fine to use Flex though.